questions_pop_culture.py

```diff
@@ -15,6 +15,9 @@
     'In "Silicon Valley" Season 3 Episode 1, what language is being used?\n@SiliconHBO',  # https://www.reddit.com/r/SiliconValleyHBO/comments/4gbra7/i_got_the_silicon_valley_s03e01_code_to_compile/)
     'What day is celebrated as Programmer\'s day? Why?\n#ProgrammersDay',  #
     '🐠In Mr. Robot, what is the name of Elliot\'s betta fish? @whoismrrobot #MrRobot',  # Qwerty
+    'Who was the first Disney Princess? @DisneyStudios' #Disney
+    'Who created Mickey Mouse? @DisneyStudios #Disney' #Disney #MickeyMouse
+    'Why is Hacktoberfest done only once in a year? @Hacktoberfest #Hacktoberfest2020'
```

Comment:
This is a great question. Remember to follow the example in issue #5 
```python
'question goes here',  # short answer goes here, add a link if it's not a short answer
```
And the below to keep in line with Python syntax
   * Each question should be in its own line
   * Surrounded by single quotes (`'`)
   * Keep the trailing comma